### PR TITLE
AKO 1.10 changes

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by akshayhavile** [comment]
build ako

**Review by swathinsankaran — DISMISSED** [review]
LGTM.

**Comment by swathinsankaran** [comment]
build ako

**Review by DixitAakash — DISMISSED** [review]

**Comment by akshayhavile** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

File: buildsettings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (swathinsankaran) [inline buildsettings.json:1]: We have to cherry-pick the changelogs to the master.

**Comment by swathinsankaran** [comment]
build ako

**Review by swathinsankaran — APPROVED** [review]

**Review by aaha97 — APPROVED** [review]
LGTM

**Pull request merged**